README.md

```diff
@@ -108,6 +108,8 @@
 #### Casual
 
 <!--lint ignore double-link-->
+- [Antimine - Minesweeper](https://github.com/lucasnlm/antimine-flutter) - Antimine is a minesweeper-like puzzle game. The objective is to flag the spaces with mines to make the field a safer place without exploding any of them. Get it on [Google Play](https://play.google.com/store/apps/details?id=com.logical.minato). By [lucasnlm](https://github.com/lucasnlm).
+
```

Comment:
Why is there an empty line needed here? 🤔

Reply:
I got a lint error on my previous commit. Let me double check.